pass-deposit-services/deposit-core/src/main/resources/application.properties

```diff
@@ -20,11 +20,13 @@ pass.client.url=${PASS_CLIENT_URL:localhost:8080}
 pass.client.user=${PASS_CLIENT_USER:fakeuser}
 pass.client.password=${PASS_CLIENT_PASSWORD:fakepassword}
 
-ftp.host=${FTP_HOST:localhost}
-ftp.port=${FTP_PORT:21}
+ftp.host=${PMC_FTP_HOST:localhost}
+ftp.port=${PMC_FTP_PORT:21}
 
 dspace.host=${DSPACE_HOST:localhost}
 dspace.port=${DSPACE_PORT:8181}
+dspace.user=${DSPACE_USER:test@test.edu}
+dspace.password=${DSPACE_PASSWORD:admin}
```

Comment:
I was expecting to see `dspace.server` added here?

Reply:
I wasn't sure if this was needed. I know when I changed some of these application properties around, I was getting errors because there is an expected set of properties that the `ProtocolBinding` is expecting. I can't remember if add dspace.server caused this error or not, or if it was another variable that I changed that threw the error.

Reply:
It should be required, the spring will set the placeholder in the repositories.json with the property value.